backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxBackend.scala

```diff
@@ -221,7 +230,8 @@ object VeloxBackendSettings extends BackendSettingsApi {
 
     val validationChecks = Seq(
       validateScheme(),
-      validateFormat(),
+      validateFormat(format),
```

Comment:
Do we still need validateFormat? Looks like  validatePartitionFormat is enough.

Reply:
I think we should keep it. Because all the partition's formats are possible not same as table's format.

Reply:
Now, I remove `validateFormat` and using `validateFormats` to check them together.